.kokoro/tests/run_tests.sh

```diff
@@ -130,18 +130,22 @@ ROOT=$(pwd)
 if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
     echo "=== Setting up DRIFT region tag injector ==="
     # install PyYaml (used by the DRIFT region tag parsing system)
-    echo "--- Installing PyYaml ---"
-    python3 -m pip install --user pyyaml
+    echo "--- Installing pip packages ---"
+    pip3 --version
+    pip3 install --user pyyaml frozendict recordclass
```

Comment:
Isn't it possible to use `python3 -m pip install --user` here and `python3` in `run_single_test.sh`?

Reply:
Not without some kludgery - I get a `ModuleNotFound` error for `frozendict` when using `python3`. Furthermore, I get `pip 20.3.3 from /usr/local/lib/python3.8/site-packages/pip (python 3.8)` when I run `pip3 --version`.

If I run `python3 -m pip install --user frozendict` (in the container's `bash` prompt, not the script itself) I get `No module named pip`.